frontend/class-twitter.php

```diff
@@ -198,6 +201,27 @@ private function single_description( $post_id = 0 ) {
 		return strip_tags( get_the_excerpt() );
 	}
 
+
+	/**
+	 * Getting the description for the taxonomy
+	 *
+	 * @return bool|mixed|string
+	 */
+	private function taxonomy_description() {
+		$meta_desc = WPSEO_Taxonomy_Meta::get_meta_without_term( 'twitter-description' );
+
+		if ( ! is_string( $meta_desc ) || $meta_desc === '' ) {
+			$meta_desc = $this->fallback_description();
```

Comment:
Because the value might be empty and in that case it wil do the stuff on line 221.